src/test/java/bio/terra/grammar/GrammarTest.java

```diff
@@ -146,8 +147,15 @@ public void testSynapseTranslate() {
         Query.parse(
             "SELECT vocabulary.datarepo_row_id FROM datasetName.vocabulary WHERE vocabulary.vocabulary_id IN ('1')");
     String expectedQuery =
-        "SELECT alias927641339.datarepo_row_id FROM (SELECT * FROM\nOPENROWSET(\n  BULK 'metadata/parquet/vocabulary/*/*.parquet',\n  DATA_SOURCE = 'sourceDatasetDataSourceName1',\n  FORMAT = 'parquet') AS inner_alias927641339) AS alias927641339\n WHERE alias927641339.vocabulary_id IN ( '1' )";
-    String translatedQuery = query.translateSql(synapseVisitor);
+        QueryTestUtils.collapseWhiteSpace(
+            """
+    SELECT alias927641339.datarepo_row_id FROM (SELECT * FROM
```

Comment:
I updated the expected sql to be more readable and then collapsed the white space so that the change is not breaking due to different formatting